tests/Unit/Domain/Test_InterfaceItems.cpp

```diff
@@ -520,5 +697,6 @@ SPECTRE_TEST_CASE("Unit.Domain.InterfaceItems", "[Unit][Domain]") {
   test_interface_subitems();
   test_interface_slice();
   test_interface_base_tags();
+  test_boundary_coordinates_moving_mesh();
```

Comment:
[nitpick] Make order of test function definitions consistent with order of test function calls... either by moving this up one line, or by moving the new test functions down under `test_interface_base_tags`.